src/lidar/lidar_metadata_in_db.py

```diff
@@ -148,10 +148,9 @@ def get_lidar_path(engine, geometry_df):
 
 
 def main():
-
     engine = setup_environment.get_database()
     Lidar.__table__.create(bind=engine, checkfirst=True)
-    file_path_to_store = r"P:/DT/LiDAR/lidar_data"
+    file_path_to_store = r"U:/Research/FloodRiskResearch/DigitalTwin/LiDAR/lidar_data"
```

Comment:
again will be great to use pathlib here in future

Reply:
done